.drone.yml

```diff
@@ -75,6 +75,11 @@ pipeline:
       - export DRONE_DIR="/drone"
       - ./bin/add-coursier.sh
       - /usr/local/bin/setup_keys.sh
+      # Setup everything to be able to push to github pages
```

Comment:
I recommend moving these sort of things into bash scripts, large .travis/.drone.yml files are a code smell IMO